docs/notebooks/inequality_constraints.pct.py

```diff
@@ -146,6 +148,105 @@ def create_bo_model(data):
 )
 plt.show()
 
+# %% [markdown]
+# ## Define a batch acquisition rule
+#
+# It is sometimes beneficial to query several points at a time instead of one. We show here how to create ad-hoc extensions of the previous acquistion function and rule.
+
+# %%
+from trieste.acquisition.function import BatchAcquisitionFunctionBuilder, BatchReparametrizationSampler
+
+
+class BatchExpectedConstrainedImprovement(BatchAcquisitionFunctionBuilder):
```

Comment:
i've been going with the following in the notebooks (at least where for things I've only used once). What do you reckon?
```suggestion
class BatchExpectedConstrainedImprovement(trieste.acquisition.BatchAcquisitionFunctionBuilder):
```